lib/TPP/Transforms/TileConsumerAndFuseProducers.cpp

```diff
@@ -574,7 +587,7 @@ static Operation *getLastFusableEltWiseConsumer(
   Value linalgOpRes = linalgOp->getResult(0);
   // If we allow use, we may end up doing recomputation. Unclear if it is
   // profitablem thus disallow for now.
-  if (!linalgOpRes.hasOneUse())
```

Comment:
Please add simple test cases that validate this change to `tile-and-fuse.mlir` and `tile-and-fuse-named-op.mlir`.